rumydata/field.py

```diff
@@ -116,7 +134,7 @@ def _comparison_columns(self) -> set:
         compares = set()
         for r in self.rules:
             if issubclass(type(r), clr.ColumnComparisonRule):
-                compares.add(r.compare_to)
+                compares.add(r.compare_to) if isinstance(r.compare_to, str) else [compares.add(x) for x in r.compare_to]
```

Comment:
Ooh that's a really nice use of an inline add. I didn't realize you could do it like that.